src/include/test/CMakeLists.txt

```diff
@@ -62,24 +62,95 @@ message(STATUS "Data files for testing will be extracted to ${SIFT_TEST_DIR}")
 # kmeans_add_test(gen_queries)
 # kmeans_add_test(time_queries)
 
+# find_package(Matplot++ REQUIRED)
+
+# kmeans_add_exe(demo_high_five)
+# target_link_libraries(demo_high_five PRIVATE HighFive)
+
+# kmeans_add_exe(demo_matplot)
+# target_link_libraries(demo_matplot PUBLIC Matplot++::matplot)
+
+# kmeans_add_exe(demo_nn-descent)
+# target_link_libraries(demo_nn-descent PUBLIC Matplot++::matplot)
+
+# kmeans_add_exe(demo_nn-graph)
+# target_link_libraries(demo_nn-graph PUBLIC Matplot++::matplot)
+
+# kmeans_add_exe(demo_vamana)
+# target_link_libraries(demo_vamana PUBLIC Matplot++::matplot)
+
+kmeans_add_exe(time_insert)
+
+kmeans_add_exe(time_abstraction_penalty)
+
+kmeans_add_test(unit_vamana)
+
+kmeans_add_test(unit_adj_list)
 
 kmeans_add_test(unit_algorithm)
 
+kmeans_add_test(unit_api)
+
+kmeans_add_test(unit_api_feature_vector)
+
+kmeans_add_test(unit_api_feature_vector_array)
+
+kmeans_add_test(unit_api_flat_l2_index)
+
+kmeans_add_test(unit_api_flat_pq_index)
+
+kmeans_add_test(unit_api_ivf_flat_index)
+
+kmeans_add_test(unit_api_ivf_pq_index)
+
+kmeans_add_test(unit_api_vamana_index)
+
+kmeans_add_test(unit_compat)
+
 kmeans_add_test(unit_concepts)
 
+kmeans_add_test(unit_concepts_vs)
+
+kmeans_add_test(unit_cpos)
+
+# kmeans_add_test(unit_old_concepts)
```

Comment:
Is this (and the others below) commented out because it's failing? If not, could we comment back in?